salt/modules/glusterfs.py

```diff
@@ -660,3 +664,82 @@ def list_quota_volume(name):
         ret[path] = _etree_to_dict(limit)
 
     return ret
+
+
+def get_op_version(name):
+    '''
+    Returns the glusterfs volume op-version
+    name
+        Name of the glusterfs volume
+    CLI Example:
+    .. code-block:: bash
+
+        salt '*' glusterfs.get_op_version <volume>
+    '''
+
+    version = ''
+    cmd = 'gluster volume get {0} cluster.op-version'.format(name)
```

Comment:
Can you change this to a list of command options, rather than a string? It's safer that way since `name` could pass in anything.

Reply:
To make sure I understand what you're asking, is this what I should change it to?

```cmd = ['gluster', 'volume', 'set', 'all', 'cluster.op-version', version]```

Reply:
Yes, that's it exactly!